js/client-chat-tournament.js

```diff
@@ -315,7 +315,8 @@
 					} else if (this.info.teambuilderFormat.substr(0, 4) === 'gen5' && !Dex.loadedSpriteData['bw']) {
 						Dex.loadSpriteData('bw');
 					}
-					this.room.$chat.append("<div class=\"notice tournament-message-start\">The tournament has started!</div>");
+					var participants = data[0] ? " (" + data[0] + " participants)" : "";
```

Comment:
```suggestion
					var participants = data[0] ? " (" + data[0] + " players)" : "";
```

"Participants" is a less common word, and probably harder for non-native speakers.